pkgs/games/aaaaxy/default.nix

```diff
@@ -40,11 +41,17 @@ buildGoModule rec {
     pkg-config
   ];
 
+  outputs = [ "out" "testing_infra" ];
```

Comment:
```suggestion
  outputs = [ "out" "tests" ];
```

Reply:
I can't name it `tests` because I already defined `passthru.tests` in the package and the two would collide. Do you have any other name suggestions?

Reply:
good catch